homeassistant/components/light/lifx.py

```diff
@@ -227,6 +227,25 @@ def turn_on(self, **kwargs):
         self._liffylights.set_color(self._ip, hue, saturation,
                                     brightness, kelvin, fade)
 
+    def set_brightness(self, **kwargs):
```

Comment:
Actually what you described will not work. set_brightness is called via

`hass.services.call(DOMAIN, SERVICE_TOGGLE, data)` afaik.

data is a key value pair containing the args from config.